aea/cli_gui/__init__.py

```diff
@@ -222,10 +220,9 @@ def search_registered_items(item_type: str, search_term: str):
 def create_agent(agent_id: str):
     """Create a new AEA project."""
     ctx = Context(cwd=app_context.agents_dir)
+    author = app_context.author if app_context.local else None
```

Comment:
why only `if app_context.local`

Reply:
Because of the logic in create method:
```
    if author is not None:
        if local:
            do_init(author, False, False)
        else:
            raise click.ClickException(
                "Author is not set up. Please use 'aea init' to initialize."
            )
```
That means that initializing default author is for local usage only (which makes sense).

Reply:
See above and remove;